MidasCivil_Adapter/Convert/ToMidasCivil/Elements/FromRigidLink.cs

```diff
@@ -58,6 +58,7 @@ public static string FromRigidLink(this RigidLink link, string version)
                 case "9.4.0":
                 case "9.4.5":
                 case "9.5.0":
+                case "9.5.0.nx":
```

Comment:
Can we add a check here for XtoYY/ZZ, YtoXX,ZZ and ZtoXX/YY please and raise a warning if they are true. Something like:
```
LinkConstraint con = link.Constraint;
if(con.XtoYY || lcon.XtoZZ || con.YtoXX || lcon.YtoZZ || con.ZtoXX || lcon.ZtoYY)
    Engine.Base.RecordWarning("Imposed rotations due to translations are not supported in this Adapter.");
```